gorm-tools/src/main/groovy/gorm/tools/problem/ProblemHandler.groovy

```diff
@@ -39,6 +41,13 @@ import yakworks.i18n.icu.ICUMessageSource
 @CompileStatic
 class ProblemHandler {
 
+    //Exceptions classes, for which we dont want log stacktraces in log files
```

Comment:
why are we doing this here and no in the logback config?

Reply:
Fixed as per discussion on slack